src/Hazelcast.Net/DistributedObjects/Impl/HMultiDictionary.cs

```diff
@@ -36,10 +37,12 @@ public HMultiDictionary(string name, DistributedObjectFactory factory, Cluster c
             _lockReferenceIdSequence = lockReferenceIdSequence;
         }
 
-        public Task<Guid> SubscribeAsync(bool includeValues, Action<MultiDictionaryEventHandlers<TKey, TValue>> handle)
+        /// <inheritdoc />
+        public Task<Guid> SubscribeAsync(Action<MultiDictionaryEventHandlers<TKey, TValue>> handle, bool includeValues = true)
```

Comment:
Should `handle` becomes `events`? Event if the method is not public... it's more consistent.

Reply:
done